### DISCUSSION TITLE
Need to show the List Name of items showed in the search results parts

### DISCUSSION THREAD

**Discussion by bastide60** [microsoft-search/pnp-modern-search#2989] [Q&A]
hi all,

I have a simple query that works to show items from different lists of my sharepoint
![image](https://github.com/microsoft-search/pnp-modern-search/assets/10298763/ea90b7a6-6b2f-4bbc-b596-54c233f11947)

But i need to get the names of the lists for each items and i don't know where to find it.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/10298763/43b03931-f660-4e0a-b4ed-d016c535afcf)

I don't see property or refiner to solve my problem.

Anyone know how can i get the Lists Names for each of my results pls ?
Thanks

**Comment by kasperbolarsen** [comment]
As I recall it you will have to map ows_ListTitle to a managed property in order to get access to the List Display Name, see https://sharepoint.stackexchange.com/questions/292959/how-to-sort-by-name-with-the-highlighted-content-web-part  for details.

Have a nice day
